### PR TITLE
Md dialog class

### PR DESCRIPTION
**Pull request by anandsinghparihar**

### CODE REVIEW

**Comment by anandsinghparihar** [comment]
https://particl.atlassian.net/browse/PAR-242

File: src/app/wallet/address-book/address-book.component.ts [hunk 1]
```diff
@@ -16,12 +16,12 @@ export class AddressBookComponent {
   }
 
   openNewAddress(): void {
-    const dialogRef = this.dialog.open(NewAddressModalComponent);
+    const dialogRef = this.dialog.open(NewAddressModalComponent, {panelClass: 'small-model'});
```
Comment (rynomster) [inline src/app/wallet/address-book/address-book.component.ts]: Is it a model or a modal?

Comment (anandsinghparihar) [reply src/app/wallet/address-book/address-book.component.ts]: Yeah, Typo mistake. it should be a modal >> **done **

**Comment by rynomster** [comment]
Shouldn't we call it dialog? Also why did we add a class all of a sudden
when before we didn't have one?

**Comment by anandsinghparihar** [comment]
Allien and CRZ want small and full-width modal class.
I am going to change into >> full-width-modal & small-modal.

**Comment by kewde** [comment]
Can't we auto-detect which class to use?
Or default to a certain value?

**Comment by anandsinghparihar** [comment]
I discussed with Allien he said ` it's no longer needed` . so please close this PR.

**Pull request closed without merging**